glide.yaml

```diff
@@ -156,6 +156,23 @@ import:
   version: c06e80d9300e4443158a03817b8a8cb37d230320
   subpackages:
   - rate
+- package: golang.org/x/text
```

Comment:
No, not "update" it,  I just specify a version.
If this line is not add to the file, every time you run `make update`, 
this package would update to the latest. @shenli